modules/web/src/app/shared/components/labels/component.ts

```diff
@@ -12,20 +12,36 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import {Component, Input, OnChanges, OnInit, SimpleChanges} from '@angular/core';
+import {AfterViewInit, ChangeDetectorRef, Component, Input, OnChanges, OnInit, SimpleChanges} from '@angular/core';
 
 @Component({
   selector: 'km-labels',
   templateUrl: './template.html',
+  styleUrls: ['style.scss'],
 })
-export class LabelsComponent implements OnInit, OnChanges {
+export class LabelsComponent implements OnInit, OnChanges, AfterViewInit {
   @Input() labels = {};
   @Input() limit: number;
   @Input() emptyMessage = '';
+  @Input() oneLineLimit = false;
   labelKeys: string[] = [];
+  showHiddenLabels = false;
+  showAndHideExtraLabels = false;
+
+  get showLabelsText(): string {
```

Comment:
I think we can handle this directly inside HTML instead of adding this getter. Wdyt?